src/Workspaces/VisualBasic/Portable/Indentation/SpecialFormattingOperation.vb

```diff
@@ -192,12 +194,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Indentation
             End If
 
             operations.Add(FormattingOperations.CreateAlignTokensOperation(
-                           cases(0).GetFirstToken(includeZeroWidth:=True),
-                           cases.Skip(1).Select(Function(n) n.GetFirstToken(includeZeroWidth:=True)),
-                           AlignTokensOption.AlignIndentationOfTokensToBaseToken))
+                cases(0).GetFirstToken(includeZeroWidth:=True),
+                cases.Skip(1).SelectAsArray(Function(n) n.GetFirstToken(includeZeroWidth:=True)),
```

Comment:
💭 After this change is complete, we could experiment with changing `AlignTokensOperation` to `AlignTokenOperation`, and splitting up any case of multiple tokens into multiple operations instead.